docs/en/cloud/AWS-CFT.md

```diff
@@ -0,0 +1,261 @@
+- An account on [AWS](https://aws.amazon.com/)
+- Web console access to [CloudFormation](https://console.aws.amazon.com/cloudformation) and [EC2](https://console.aws.amazon.com/ec2)
+- A terminal window with [SSH](https://man.openbsd.org/ssh.1)
+
+Familiarity with Cloudformation and EC2 is helpful but not required.
+The tutorial launches an Alluxio cluster from the perspective of a user with a newly created AWS account.
+
+Note that the launched instances do not qualify for free usage tier.
+The default instance type we will be using is **r4.4xlarge**. 
+
+## Launch an Alluxio Cluster
+
+This section outlines how to launch an Alluxio cluster.
+
+### Create an EC2 key pair
+
+If you have valid EC2 key pair already, skip ahead to the next step.
+
+{% accordion keypair %}
+  {% collapsible Create an EC2 key pair %}
+CloudFormation will analyze existing AWS resources on your account to fill in values needed by the Alluxio CloudFormation template (CFT).
+For example, you need to choose an existing EC2 key pair that is in the region in which you are creating the Alluxio cluster stack.  
+
+Open a web browser and navigate to the EC2 console, logging in with your credentials when prompted.
+Note the AWS region in the upper right corner; click on the region to open a dropdown of available regions.
+It is recommended to select the region that is geographically closest to your computer.
+
+On the left sidebar, click the Key Pairs link to view created key pairs for this region.
+Click the Create Key Pair button to create a new key pair and download the it to your computer. 
+
+Assuming the downloaded key pair is located at `~/Downloads/keypair.pem`,
+change the file permissions of the key pair to read-only by running the following command:
+```console
+chmod 400 ~/Downloads/keypair.pem
+```
+
+  {% endcollapsible %}
+{% endaccordion %}
+
+### Choose the CloudFormation template to launch
+
+{% accordion launch %}
+  {% collapsible Choose the CloudFormation template to launch %}
+Navigate to the [CloudFormation console](https://console.aws.amazon.com/cloudformation) and click Create Stack.
+We have two templates available: a basic and an advanced version.
+The advanced version is a copy of the basic one with additional parameters and configuration options exposed when creating the stack.
+
+Copy either URL in the *Amazon S3 URL* field:
+- Basic: [https://alluxio-public.s3.amazonaws.com/cft/{{site.ALLUXIO_RELEASED_VERSION}}/basic.json](https://alluxio-public.s3.amazonaws.com/cft/{{site.ALLUXIO_RELEASED_VERSION}}/basic.json)
+- Advanced: [https://alluxio-public.s3.amazonaws.com/cft/{{site.ALLUXIO_RELEASED_VERSION}}/advanced.json](https://alluxio-public.s3.amazonaws.com/cft/{{site.ALLUXIO_RELEASED_VERSION}}/advanced.json)
+
+  {% endcollapsible %}
+{% endaccordion %}
+
+### Configure cluster
+
+{% accordion configureCluster %}
+  {% collapsible Configure cluster %}
+Specify the details of your Alluxio cluster in this page:
+- Stack details: Name of your cluster
```

Comment:
Stack details is the page of holding all the CFT parameters.
The first value that need to put in is `Stack Name`: Name of your cluster.